qa/zcash/smoke_tests.py

```diff
@@ -351,11 +348,11 @@ def transaction_chain(zcash):
         if not zcash.validateaddress(chain_end)['isvalid']:
             print('Invalid transparent address')
             return results
-        print()
-        print('Please send at least 0.01 ZEC/TAZ to the following address:')
-        print(taddr_1)
-        print()
-        input('Press Enter once the funds have been sent.')
+
+        print('\nPlease send at least 0.01 ZEC/TAZ to the following address:')
+        print(sprout_zaddr_1)
```

Comment:
The merge from master took the old code instead of the new code (we can't send into the Sprout pool anymore). Please rebase the PR and undo this change (this entire part of the diff should go away, per my earlier comment about newlines).